### PR TITLE
[O2B-639] Add new info from Core to Run-Details

### PR DESCRIPTION
**Pull request by JikHuijberts**
#### I have a JIRA ticket
- [x] branch and/or PR name(s) include(s) JIRA ID
- [x] issue has "Fix version" assigned
- [x] issue "Status" is set to "In review"
- [x] PR labels are selected
- [x] `CHANGELOG.md` and `database/CHANGELOG.md` files were updated

Run details now show the extra fields stated below:
- Trigger Global Run Enabled;
- Trigger Enabled;
- PDP Configuration Option;
- PDP Topology Description Library File;
- TFB DD Mode;

And created a default value that can be set for `null` values.

### CODE REVIEW

File: lib/public/utilities/formatTimestamp.js [hunk 1]
```diff
@@ -43,3 +43,21 @@ export const formatTimestamp = (timestamp, inline = true) => {
...
+ */
+export const formatBoolean = (boolean) => {
+    if (boolean != null) {
+        if (boolean) {
+            return 'On';
+        }
+        return 'Off';
+    }
+    return '-';
+};
```
Comment (martinboulais) [inline lib/public/utilities/formatTimestamp.js:63]: Can you please put this in its own file?
I think in general the best is one exported function => one file, with the same name as the exported function

File: lib/public/components/RunDetail/index.js [hunk 2]
```diff
@@ -122,19 +142,19 @@ const activeFields = (model) => ({
         name: 'Data Distribution (FLP)',
         visible: true,
         size: 'cell-s',
-        format: (boolean) => boolean ? 'On' : 'Off',
+        format: (boolean) => formatBoolean(boolean),
```
Comment (martinboulais) [inline lib/public/components/RunDetail/index.js:145]: simply give the function reference as property, no need to wrap in a function (same for the others)

**Review by martinboulais — CHANGES_REQUESTED** [review]

File: lib/public/components/Detail/index.js [hunk 3]
```diff
@@ -19,14 +19,25 @@ import { h } from '/js/src/index.js';
...
+    key, {
+        name,
+        format,
+    },
+]) => h(
+    `.flex-row.justify-between#${selectorName}-${key}`,
+    h('b', `${name}:`),
+    (()=> {
+        if (format) {
+            return format(post[key]);
```
Comment (martinboulais) [inline lib/public/components/Detail/index.js:35]:
```suggestion
            return format(post[key], post);
```

**Review by martinboulais — CHANGES_REQUESTED** [review]

File: lib/public/components/Detail/index.js (hunk 3 above)
Comment (JikHuijberts) [reply lib/public/components/Detail/index.js]: Fixed it thanks for the tip!

File: lib/public/components/RunDetail/index.js (hunk 2 above)
Comment (JikHuijberts) [reply lib/public/components/RunDetail/index.js]: done!

File: lib/public/utilities/formatTimestamp.js (hunk 1 above)
Comment (JikHuijberts) [reply lib/public/utilities/formatTimestamp.js]: Done!

**Review by martinboulais — APPROVED** [review]

**Pull request merged**